modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -50,19 +61,35 @@ function perflab_aea_audit_enqueued_styles() {
 	if ( ! is_admin() && is_front_page() && current_user_can( 'view_site_health_checks' ) && false === get_transient( 'aea_enqueued_front_page_styles' ) ) {
 		global $wp_styles;
 		$enqueued_styles = array();
-		foreach ( $wp_styles->queue as $handle ) {
-			$src = $wp_styles->registered[ $handle ]->src;
-			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+		foreach ( $wp_styles->done as $handle ) {
+			$style = $wp_styles->registered[ $handle ];
+			if ( $style->src && ! strpos( $style->src, 'wp-includes' ) ) {
+
+				// Check if we already have the style's path ( part of a refactor for block styles from 5.9 ).
+				if ( ! empty( $style->extra ) && ! empty( $style->extra['path'] ) ) {
+					$path = $style->extra['path'];
+				} else { // Fallback to getting the path from the style's src.
+					$path = perflab_aea_get_path_from_resource_url( $style->src );
+				}
```

Comment:
By inverting the conditional here allows the removal of the `!` operator, allowing for a more natural flow.

```suggestion
				if ( empty( $style->extra ) || empty( $style->extra['path'] ) ) {
					$path = perflab_aea_get_path_from_resource_url( $style->src );				
				} else { // Fallback to getting the path from the style's src.
					$path = $style->extra['path'];
				}
```

Reply:
@mitogh Not sure I agree in this case, the original code here explicitly looks at whether the `$style->extra` value is not empty, and the _other_ function use would be the fallback. I don't have a strong preference on the order, but with your suggestion here the comment in line 71 is no longer accurate.

Reply:
Oh, right let me update line 71.

Mostly this pattern avoids the usage of `!` which translates 1 operation into 2.

Reply:
I'd argue that's a bit of a micro optimization. IMO the code path checking `! empty( $var )` and then using `$var` is more straightforward to read.

Reply:
The suggestion goes more towards optimization for reading rather than performance so it becomes a personal preference I would say either way is fine, but IMO this seems easier to read:

- If empty then
- Else

Rather than 

- If not empty then
- else

Reply:
I also don't agree with this (since is a personal preference), to me, seems more natural and easier to read the way it is right now.